tests/app/test_model.py

```diff
@@ -227,6 +231,24 @@ def test_letter_notification_serializes_with_subject(client, sample_letter_templ
     assert res['subject'] == 'Template subject'
 
 
+def test_user_service_role_serializes_without_updated(client, sample_user_service_role, sample_user, sample_service):
+    res = sample_user_service_role.serialize()
+    assert res['id'] is not None
```

Comment:
I won't request changes, but "res" (short for "resource," I assume) isn't the most descriptive name.  I would burn the keystrokes and call it "user_service_role".

Reply:
I'm pretty sure that was the idea was that `res` == resource or something along these lines. I was keeping it consistant with the other `.serialize()` tests in the file. 

Coming from other languages that have way more robust naming conventions this did bother me a little bit. ROFL. Objective-C and Swift both have REALLY long naming standards and grammar rules to be as descriptive as possible. Dart did something similar. Javascript, not as much. It's interesting to see how different languages lay out a best practice around these things.